packages/checkbox/src/CheckboxEditor.tsx

```diff
@@ -86,29 +99,41 @@ export function CheckboxEditor(props: CheckboxEditorProps) {
           setValue(newValues);
         };
 
+        const mergedOptions = getMergedOptions(values, options);
+
         return (
           <Form
             testId="checkbox-editor"
             spacing="condensed"
             className={cx(styles.form, direction === 'rtl' ? styles.rightToLeft : '')}>
-            {options.map((item) => (
-              <CheckboxField
-                key={item.id}
-                labelIsLight
-                id={item.id}
-                checked={values.includes(item.value)}
-                labelText={item.label}
-                disabled={disabled}
-                value={item.value}
-                name={`${field.id}.${field.locale}`}
-                onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                  if (e.target.checked) {
-                    addValue(item.value);
-                  } else {
-                    removeValue(item.value);
-                  }
-                }}
-              />
+            {mergedOptions.map((item) => (
+              <div key={item.id}>
+                <CheckboxField
+                  key={item.id}
+                  labelIsLight
+                  id={item.id}
+                  checked={values.includes(item.value)}
+                  labelText={item.label}
+                  disabled={disabled}
+                  value={item.value}
+                  name={`${field.id}.${field.locale}`}
+                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                    if (e.target.checked) {
+                      addValue(item.value);
+                    } else {
+                      removeValue(item.value);
+                    }
+                  }}
+                />
+                {item.id.includes('invalid') && (
+                  <>
+                    <span className={styles.invalidText}>(invalid)</span>
```

Comment:
I'm not sure if this is a sufficient explanation for the customer. It's not actionable and I don't understand why it's invalid even from the code :)

Reply:
Original problem is when a user removes an option from a valid set of values (e.g. must be one of 'one', 'two', or 'three'). When an entry has that value there is currently no way in the checkbox view to remove that invalid value

Reply:
Let's consult with a designer (@joaoramos) about the copy. 

One alternative solution would be to call `removeValue` automatically in case of misconfiguration.

Reply:
I'd rather be explicit to the customer about it. @dominicbonnice is driving the effort from Pulitzer